packages/big-design/src/components/Worksheet/types.ts

```diff
@@ -8,7 +10,8 @@ export interface Worksheet<Item extends WorksheetItem> {
 export interface WorksheetColumn<Item> {
   hash: keyof Item;
   header: string;
-  type?: 'text' | 'number';
+  options?: SelectOption<unknown>[];
+  type?: 'text' | 'number' | 'select';
```

Comment:
Should I try to split this type into two since ideally we only need `options` when `type === select`? What would be the best approach?

Reply:
Yeah, may be better to split the interface into two, maybe `WorksheetTextColumn` & `WorksheetSelectableColumn`? Thoughts?

Reply:
That is what I had in mind, let me try it out.